content_script.js

```diff
@@ -1,32 +1,33 @@
-/** 
+/**
   * Class Schedule to ICS
-  * (c) 2013 Keanu Lee
+  * (c) 2013-2015 Keanu Lee
+  * (c) 2015 Baraa Hamodi
   *
   * Get ICS files for university class schedules in Oracle PeopleSoft systems (including UW Quest)
-  * 
+  *
   * TODO list:
   * - Properly handle date numbers (e.g. 20130901 - 1 != 20130900, 20130131 + 1 != 20130132)
 **/
 
 $(function() {
-
   // Ontario is UTC-4
+  // TODO: Create a getUTCOffset function that handles day light savings.
   utc_offset  = -4;
 
 
   // 05/17/1992 -> 19920517
   function getDateString(date) {
-    return date.substr(6,4)
-      + date.substr(0,2)
-      + date.substr(3,2);
+    return date.substr(6,4) +
+            date.substr(0,2) +
```

Comment:
The code uses two space indentation - subsequent lines after `return` should be indented only two spaces in:

```
return ...
  ...
```